submissions/kovalov/html-css-popup/README.md

```diff
@@ -0,0 +1,3 @@
+# dropdown-menu-css
```

Comment:
Remove files not for review: images, compiled files, configs, add only source code